frontend/src/components/layout/Header.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2022 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file https://github.com/redpanda-data/redpanda/blob/dev/licenses/bsl.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+import { ChevronRightIcon } from '@primer/octicons-react';
+import { Route as AntBreadcrumbRoute } from 'antd/lib/breadcrumb/Breadcrumb';
+import { PageHeader } from 'antd';
+import { observer } from 'mobx-react';
+import { NavLink } from 'react-router-dom';
+import { isEmbedded } from '../../config';
+import { uiState } from '../../state/uiState';
+import { MotionDiv } from '../../utils/animationProps';
+import { ZeroSizeWrapper } from '../../utils/tsxUtils';
+import { UserPreferencesButton } from '../misc/UserPreferences';
+import DataRefreshButton from '../misc/buttons/data-refresh/Component';
+
+const AppPageHeader = observer(() => {
+
+    const breadcrumbs = uiState.pageBreadcrumbs.map(v => ({ path: v.linkTo, breadcrumbName: v.title }));
+    const selectedClusterName = uiState.selectedClusterName;
+    if (selectedClusterName) {
+        //const rootBreadcrumb: AntBreadcrumbRoute = { path: '', breadcrumbName: selectedClusterName };
+        const rootBreadcrumb: AntBreadcrumbRoute = { path: '', breadcrumbName: 'Cluster' };
+        breadcrumbs.unshift(rootBreadcrumb);
+    }
+
+    if (isEmbedded())
+    breadcrumbs.splice(0, breadcrumbs.length - 1);
```

Comment:
[boulder] This should be in a single line to avoid confusion
```suggestion
    if (isEmbedded()) breadcrumbs.splice(0, breadcrumbs.length - 1);
```